pkg/daemon/ceph/osd/volume.go

```diff
@@ -497,6 +511,9 @@ func (a *OsdAgent) initializeDevices(context *clusterd.Context, devices *DeviceO
 			rawDevices.Entries[name] = device
 			continue
 		}
+		if isSupportOnlyRawMode(device, &a.storeConfig) {
```

Comment:
Instead of a `continue`, what about renaming the helper method like this? It is more understandable to me.
```suggestion
		if lvmModeAllowed(device, &a.storeConfig) {
			lvmDevices.Entries[name] = device
		}
```

Reply:
@travisn  Sure, this change makes the code clearer. Thanks!